src/workflows-app/components/TroubleshootingBox.ts

```diff
@@ -0,0 +1,99 @@
+import { div, h, span } from 'react-hyperscript-helpers';
+import { ClipboardButton } from 'src/components/ClipboardButton';
+import { Link } from 'src/components/common';
+import { icon } from 'src/components/icons';
+import colors from 'src/libs/colors';
+import * as Nav from 'src/libs/nav';
+import { LogTooltips } from 'src/workflows-app/utils/task-log-utils';
+
+type TroubleshootingBoxProps = {
+  name: string;
+  namespace: string;
+  logUri: string;
+  submissionId: string;
+  workflowId: string;
+  showLogModal: (modalTitle: string, logsArray: any, tesLog: string) => void;
+  executionDirectory: string;
+};
+
+export const TroubleshootingBox: React.FC<TroubleshootingBoxProps> = (props) => {
```

Comment:
Nit: generally, Terra UI components follow this convention:

```suggestion
export const TroubleshootingBox = (props: TroubleshootingBoxProps): ReactNode => {
```

In older versions of React, FC added a children prop that might not be relevant for the component.